### DISCUSSION TITLE
Prefix stripped from Navbar items in multilingual mode

### DISCUSSION THREAD

**Discussion by iwilltry42** [adityatelange/hugo-PaperMod#967] [Q&A]
Hi there!
Thanks for this amazing theme!

Recently, I tried to translate my page https://thklein.io to German.
I enabled the language switcher and it works just fine switching the language.
However, this happens:
- I'm on the homepage
- I switch language to German -> site switches to German mode (e.g. some text in the footer is German now)
- I select "Portfolio" from the Navbar menu -> it takes me to https://thklein.io/de/portfolio as expected
- BUT: Footer text is in English again
- BUT: Links in the navbar now link to the default page without /de in front
- Also, the language switcher will always bring me back to the homepage instead of staying on the current site.

Site source is here: https://github.com/iwilltry42/thklein.io
There are some customizations added recently (also a forked theme of yours added recently), but the issue was the same before as it is now.

I'm sure I'm missing something here, so any help would be appreciated :)

**Comment by adityatelange** [comment]
I've to investigate, no such thing happens on demo website 🤔
